proto/p4/p4runtime.proto

```diff
@@ -382,10 +382,28 @@ message Uint128 {
 message SetForwardingPipelineConfigRequest {
   enum Action {
     UNSPECIFIED = 0;
+    // Verify that the target can implement the given config. Do not modify the
+    // forwarding state in the target. Returns error if config is not provided.
     VERIFY = 1;
+    // Verify that the target can implement the given config and save it. Do not
```

Comment:
"the given config, and if so, save the config"
(currently it may seem like it is verifying the ability to "save it" as well).